### DISCUSSION TITLE
Welcome to shadcn-svelte Discussions!

### DISCUSSION THREAD

**Discussion by huntabyte** [huntabyte/shadcn-svelte#82] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HogarStrashni** [comment]
First of all, great work! It's a great initiative to implement the good work of `shadcn` for svelte and hopefully I will be able to contribute!
I have one possibly dumb question however: why are prop names declared in svelte components mostly prefixed with `root`? 
So if I want to use the `Switch` component for example, I need to bind the state value to `rootChecked` instead of just `checked` property:

```
<script lang="ts">
    import Switch from '$components/ui/switch/Switch.svelte';
    let isDarkModeOn = false;
</script>

<!-- this doesn't work -->
<Switch class="mr-8" bind:checked={isDarkModeOn} />

<!-- but this works -->
<Switch class="mr-8" bind:rootChecked={isDarkModeOn} />
```

Thank you in advance for your time and answer!
